azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -291,7 +308,16 @@ def process_top_levels(
         assert level_2["folders"] == ["billing"]
         assert not level_2["files"]
 
-    return namespace, clusterconfig_namespace
+    if lnm_namespace:
+        logger.debug("Determined the following namespaces:")
+        logger.debug(f"AIO namespace: {namespace}")
+        logger.debug(f"LNM svclb namespace: {lnm_namespace}")
+        # remove empty lnm svclb related folders
+        level_1 = walk_result.pop(path.join(BASE_ZIP_PATH, lnm_namespace))
+        assert level_1["folders"] == ["lnm"]
+        assert not level_1["files"]
+
+    return namespace, clusterconfig_namespace, lnm_namespace
```

Comment:
if there will be more namespace folders, we will need to move away from the tuple structure

Reply:
going to do namedtuple for now - maybe we can move to dict at some point